recipe/meta.yaml

```
@@ -65,26 +68,33 @@ source:
   - url: https://github.com/python/cpython-bin-deps/archive/nasm-2.11.06.zip         # [win]
     folder: externals/nasm-2.11.06                                                   # [win]
     sha256: de3c87b26a80e789986d8e6950c6304175d3829afe9c6c7211eb7257266ab0ac         # [win]
-  - url: https://github.com/python/cpython-source-deps/archive/libffi-3.3.0-rc0-r1.zip  # [win]
-    folder: externals/libffi-3.3.0-rc0-rc1                                              # [win]
-    sha256: 23b4347b347cb92fbfa312c95c8f9d1c8ee4416bd5cfc0f318fa01de220aa869            # [win]
-  - url: https://github.com/python/cpython-bin-deps/archive/1cf06233e3ceb49dc0a73c55e04b1174b436b632.zip  # [win]
+
+  - url: https://github.com/python/cpython-bin-deps/archive/libffi.zip                  # [win]
     folder: externals/libffi                                                            # [win]
-    sha256: 6e0c1352ec912935481de3d9272c458e284c239e5f313ff74bcdb6aa2f56be8f            # [win]
+    sha256: 57180e18749ad5e8727ae1f488621768a58808ac387968068c563bee8b2b23ec            # [win]
 
 
 build:
-  number: 5
+  number: 6
   # Windows has issues updating python if conda is using files itself.
   # Copy rather than link.
   no_link:
     - DLLs/_ctypes.pyd  # [win]
   script_env:
     - python_branding
+#   - PY_INTERP_LINKAGE_NATURE
   skip: true  # [win and vc != '14.1']
   # We delete the shared libraries.
   ignore_run_exports:   # [unix]
     - bzip2             # [unix]
+  detect_binary_files_with_prefix: False
+  # match python.org compiler standard
+  skip: True            # [win and vc != 14]
```

Should be less than 14, not != 14
```suggestion
  skip: True            # [win and vc < 14]
```

---

Whoops!